Homework_11/src/Main.java

```diff
@@ -0,0 +1,57 @@
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class Main {
+
+    public static void main(String[] args) {
+        //Creating 2 performers
+        CircusPerformer performer1 = new CircusPerformer("Ivan P.", "juggler", 10);
+        CircusPerformer performer2 = new CircusPerformer("Andrii R.", "acrobat", 15);
+        CircusPerformer performer3 = new CircusPerformer("Mykyta D.", "clown", 4);
+        CircusPerformer performer4 = new CircusPerformer("Annd F.", "Dancer", 8);
+        //try to add performer with invalid experience
+        //CircusPerformer performer5 = new CircusPerformer("dancer", "Jack Dany", -1);
+        //Adding performers to the circle
+        Circus<CircusPerformer> circus = new Circus<>();
+        circus.addPerformer(performer1);
+        circus.addPerformer(performer2);
+        circus.addPerformer(performer3);
+        circus.addPerformer(performer4);
+        // circus.addPerformer(performer5);
+        //Saving performers to the file
+        circus.saveToFile("Performers.txt");
+        //Removing performers to check further upload
+        circus.removeAllPerformers();
```

Comment:
It will work, although you're using `performers.add((T) performer);` when loading from the file, so it depends on your decision: whether we clear the Circus from existing performers before the import or we let the user risk it and deal with duplicated in case they were present there

If choice is to clear before the import -> then `removeAllPerformers()` might be called in `loadFromFile()` method just before the import -> then client won't care about it